tests/console/perl_bootloader.pm

```diff
@@ -20,16 +19,36 @@ use version_utils 'is_sle';
 sub run {
     my ($self) = @_;
     select_serial_terminal;
+    zypper_call 'in perl-Bootloader';
 
-    if (script_run 'rpm -q perl-Bootloader' == 1) {
-        zypper_call 'in perl-Bootloader';
+    if (is_sle(">=15-SP3")) {
+        assert_script_run 'pbl --loader grub2';
```

Comment:
This doesn't really make sense. Either grub2 or grub2-efi will work, but not both. It might install it and boot anyway, but using the previously installed grub and not what this test did.

Reply:
I modified it to include conditions checking whether it's uefi or not.